content/en/docs/reference/controllers/admission-pod-preset.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Pod preset controller
+content_template: templates/concept
```

Comment:
@sftim, general comment for all pages:
I think the title text should capitalize the first letter of each major word. Also, the titles should be consistently named, contain `Controller` in the title?, and closely match the actual controller name?

Reply:
I looked at https://kubernetes.io/docs/contribute/style/ and didn't see a clear steer on whether or not to capitalize titles.